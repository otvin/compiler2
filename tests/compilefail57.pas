program compilefail57(output);
{test reference outside of array bound, to the greater than side}

type myarray = array[1..9,2..12] of char;
var myarr:myarray;

begin

	myarr[1,8] := 'a';  {should work}
	myarr[4,13] := 'z';  {should compile error}

	writeln(myarr[1,8]);
end.
