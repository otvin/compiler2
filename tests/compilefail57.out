Array Index '13' out of range in: file:tests/compilefail57.pas line:10    column:13 
